Octokit/Models/Response/CommitActivity.cs

```diff
@@ -13,13 +14,15 @@ public class CommitActivity
 
         public CommitActivity(IEnumerable<WeeklyCommitActivity> activity)
         {
-            Activity = activity;
+            Ensure.ArgumentNotNull(activity, "activity");
```

Comment:
I guess I should've added these to most (all?) parameters in the other response class ctors as well, but wanted to keep to the issue at hand. Do we care?